pkg/nsx/services/common/policy_tree.go

```diff
@@ -456,8 +456,8 @@ func (b *PolicyTreeBuilder[T]) UpdateMultipleResourcesOnNSX(objects []T, nsxClie
 			return err
 		}
 		if err = nsxClient.OrgRootClient.Patch(*orgRoot, &enforceRevisionCheckParam); err != nil {
-			log.Error(err, "Failed to delete multiple resources on NSX with HAPI", "resourceType", b.leafType)
 			err = util.TransNSXApiError(err)
+			log.Error(err, "Failed to delete multiple resources on NSX with HAPI", "resourceType", b.leafType)
```

Comment:
Do we need to remove` model.SharedResource` caes handle from getNSXResourceId and getNSXResourcePath

Reply:
It looks like we keep cleanupInfraSharedResources function, if that, we need to keep `model.SharedResource` caes handle

Reply:
we just remove the cleanupInfraSharedResources but keep the logic handling the resource.